### PR TITLE
Add optional binary argument to start command, increase max-sql-memory to 10%

### PR DESCRIPTION
**Pull request by richardwu**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -127,16 +127,29 @@ stopping and wiping of clusters along with running load generators.
 `,
 }
 
+const defaultBinary = "./cockroach"
```
Comment (petermattis) [inline main.go]: Might be better to make this a flag so that it works with `test` as well.

File: cockroach.go [hunk 2]
```diff
@@ -27,6 +27,7 @@ func (r cockroach) start(c *cluster) {
 		} else {
 			args = append(args, "--insecure")
 		}
+		args = append(args, "--max-sql-memory=1GB")
```
Comment (petermattis) [inline cockroach.go]: Any reason you can't pass this via `start -a=--max-sql-memory=1GB`?

Also, for my OCD, this belongs down near the `--cache` flag.

Comment (richardwu) [reply cockroach.go]: Ah didn't notice that flag, I thought we might as well bump this default up since `roachprod` provisions 15GB memory VMs.

Comment (petermattis) [reply cockroach.go]: Probably better to do something like `--max-sql-memory=10%`.

Comment (richardwu) [reply cockroach.go]: Good point.

File: main.go (hunk 1 above)
Comment (richardwu) [reply main.go]: Done.

**Pull request merged**